tech/languages/rust/rust-sig.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Rust SIG
+subsection: rust
+order: 5
+---
+
+# Rust SIG
+
+Rust Special Interest Group takes care of anything related to Rust in Fedora distribution. You can find more information at the official web pages:
+https://docs.pagure.org/fedora-rust.sig/index.html
```

Comment:
Great!
But please make this a Markdown link (with a text, instead of just an url).